docs/model/grant.md

```diff
@@ -1,3 +1,50 @@
+A **Grant** entry captures essential details about funded research projects. Grants support various scientific studies, often involving multiple institutions, investigators, and funding agencies. Accurate grant records facilitate resource tracking, collaboration, and compliance with funding requirements.
+
+This section outlines how to create a valid grant entry by defining key metadata, including grant type, institution, investigator, and project duration. Proper grant documentation ensures projects can be easily identified, referenced, and managed across research initiatives.
+
+### **Why You Should Contribute Grant Entries**
+Contributing grant entries ensures that funding sources, research objectives, and project details are transparent and traceable. This helps stakeholders monitor the progress of research initiatives, improves collaboration by connecting teams with similar funding, and supports compliance with reporting and accountability requirements. Accurate grant documentation also strengthens your ability to secure future funding by showcasing project outcomes and impacts.
+
+#### **Who Should Be Contributing Grant Entries?**
```

Comment:
Currently, grant information is actually only curated by the MC2 Center. We are not currently asking anyone external to provide this information, but it's possible we may in the future. I am wondering if we can hide this content for now, tweak this, or actually embed this within our internal grant curation SOP (to be developed with DevDocs)